home/storage/LocalStorage.js

```diff
@@ -49,7 +49,14 @@ async function updateSettingsAsync(updatedSettings) {
 }
 
 async function getSessionAsync() {
-  let results = await AsyncStorage.getItem(Keys.Session);
+  let results = await ExponentKernel.getSessionAsync();
+  if (!results) {
+    // try querying legacy location
```

Comment:
consider including today's date in the comment so that a yearish from now we can delete this without interrupting active developers